benches/benches.rs

```diff
@@ -23,50 +24,57 @@ extern crate test;
 use noise::{perlin2, perlin3, perlin4, simplex2, simplex3, simplectic2, simplectic3, simplectic4, Seed};
 use test::Bencher;
 
+fn black_box<T>(dummy: T) -> T {
```

Comment:
I was thinking something along the lines of `This is used to prevent LLVM from optimising away the benchmarks`. The inner comment is a little obscure to me, at least from my perspective.